pkg/sync/config.go

```diff
@@ -220,6 +222,10 @@ func NewConfigFromCli(c *cli.Context) *Config {
 		logger.Warnf("threads should be larger than 0, reset it to 1")
 		cfg.Threads = 1
 	}
+	if (cfg.CheckAll || cfg.CheckNew) && cfg.CheckChange {
```

Comment:
we can do the both, check the change first, only check data later (don't update the metrics twice)